sklearn/tree/_criterion.pyx

```diff
@@ -24,8 +24,11 @@ from libc.math cimport fabs
 
 import numpy as np
 cimport numpy as np
+import random #added by morgan
```

Comment:
"#added by morgan" Are these necessary?

Reply:
No, they aren't. Thanks for reminding me. I was trying to keep track of lines I added incase issues arose later. I'll be sure to remove them.

Reply:
no problem! I know the sklearn folks will probably have you remove them anyways down the line. But a good tool to use for keeping track of your stuff you added is checking the PR "files changed", as well as using "git diff".